Tests/NooraTests/Utilities/TerminalTextTests.swift

```diff
@@ -15,6 +15,17 @@ struct TerminalTextTests {
         // Then
         #expect(got == "Please run 'tuist project tokens create' to obtain a new token.")
     }
+    
+    @Test func linksAreNotFormattedWhenNonInteractiveTerminal() {
+        // Given
+        let subject: TerminalText = "Visit \(.link(title: "Tuist", href: "https://tuist.dev"))"
+        
+        // When
+        let got = subject.formatted(theme: .default, terminal: Terminal(isInteractive: false, isColored: true))
+
+        // Then
+        #expect(got == "Visit <Tuist: https://tuist.dev>")
```

Comment:
I'm not too opinionated about this formatting. If you have other ideas let me know.

Reply:
I was also considering the markdown format [title](https://link) which a lot of devs might be familiar with. But I'm good with either.